snippets/model.cson

```diff
@@ -0,0 +1,65 @@
+'.source.coffee':
+
+
+  'available in model':
+    'prefix': 'model--help'
+    'body':
+      """
+      ${1:
+        # you can use these snippets in a Model function.
+        #
+        # DS.Model.extend(
+        #   example : att<tab>
+        #    __________________________________________________________
+        #   | type shorthand -> or long version ->  press tab to see:   |
+        #   |  -----------   ->  -------------  ->   ------------------ |
+        #   |      att       ->   ds.attr       ->   DS.attr(...        |
+        #   |      bel       ->   ds.belongsTo  ->   DS.belongsTo(...   |
+        #   |      has       ->   ds.hasMany    ->   DS.hasMany(...     |
+        #   |___________________________________________________________|
+        # )
+        #           *press <Backspace> to remove this quick guide.
+      }
+      """
+
+
+
+
+  'Ember Model':
+    'prefix': 'model'
+    'body':
+      """
+        _export = DS.Model.extend(
+          ${1:# see more snippets by Expanding :
+
+            # model--help${2}
+            # or model-h for short.
+            }
+        )
+      """
+
+
+
+
+  'Ember Model attributes':
+    'prefix': 'ds.attr'
+    'body':
+      """
+        ${1}: DS.attr(  "string"  )
```

Comment:
it just a way to remind what types are available there.
I usually suspect weather to use "number" or "numeric" or "integer" or any other keyword.